doc/releases/release-notes-3.0.rst

```diff
@@ -2,8 +2,8 @@
 
 .. _zephyr_3.0:
 
-Zephyr 3.0.0 (Working draft)
-############################
+Zephyr 3.0.0
+#############
```

Comment:
```suggestion
Zephyr 3.0.0
############
```

Reply:
dude...

Reply:
less is bad, but more has never been a problem :)

Reply:
sure, but it looks nicer that way :)